nsqadmin/static/js/app_state.js

```diff
@@ -17,7 +17,11 @@ var AppState = Backbone.Model.extend({
         this.on('change:graph_interval', function(model, v) {
             localStorage.setItem('graph_interval', v);
         });
-        this.set('graph_interval', localStorage.getItem('graph_interval') || 'off');
+
+        var qp = _.object(_.compact(_.map(window.location.search.slice(1).split('&'),
+            function(item) { if (item) { return item.split('='); } })));
+        var interval = qp['t'] || localStorage.getItem('graph_interval') || 'off';
```

Comment:
Let's just support them all and not care? `(s|seconds?|min|minutes?|h|hours?|d|days?|w|weeks?|mon|months?|y|years?)` seems to be the valid list (despite the [docs](http://graphite.readthedocs.org/en/latest/render_api.html#from-until) only showing a subset)